lib/worker.js

```diff
@@ -208,29 +208,14 @@ class AssetComputeWorker {
             return;
         }
 
-        // 2. check and log resulting rendition
+        // check and log resulting rendition
         this.timers.callbackProcessing += callbackTimer.end();
         console.log(`worker() callback generated rendition in ${callbackTimer.durationSec()} seconds: ${rendition.name}`);
 
-        // 3. post processing for images
-        // start post-processing timer
-        const postProcessingTimer = this.timers.currentPostProcessing = new Timer();
-        const shouldDoPostProcess = await this.shouldPostProcess(rendition);
-        if (shouldDoPostProcess) {
-            try {
-                rendition = await this.doPostProcess(rendition);
-                console.log(`Postprocessing finished in ${postProcessingTimer.durationSec()} seconds: ${rendition.name}`);
-            } catch (err) {
-                console.error(`rendition post-processing failed with error after ${postProcessingTimer.durationSec()}: ${err.message || err}`);
-                this.timers.postProcessing += postProcessingTimer.duration(); // add post processing rendition time to total post processing duration
-                await this.renditionFailure(rendition, err);
-                return; // continue with next rendition
-            }
-        } else {
-            console.log(`No postprocessing done for rendition ${rendition.name}`);
+        rendition = await this.postProcess(rendition);
```

Comment:
`postProcess` does not return something in all cases (if postprocessing is successful, there is no object/value returned. So the condition below will evaluate to true, always). At least I didn't see a return statement?

Reply:
See above.